CHANGES

```diff
@@ -6,6 +6,9 @@ Version 0.12.1
 
 yet to be released
 
+- Add condition of not being under Windows to reuse the file descriptor 
```

Comment:
Could we change this entry to be a bit more user-oriented? I.e. write "fix crash of reloader on Windows" (with the exact error msg) instead of what we changed within the codebase